.github/workflows/multi-arch-test-build.yml

```diff
@@ -10,28 +12,50 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - arch: arm_cortex-a9_vfpv3-d16
+            target: mvebu-cortexa9
+            runtime_test: false
+
           - arch: mips_24kc
             target: ath79-generic
             runtime_test: false
 
+          - arch: powerpc_8540
+            target: mpc85xx-p1010
+            runtime_test: false
+
+          - arch: aarch64_cortex-a53
+            target: mvebu-cortexa53
+            runtime_test: true
+
+          - arch: arm_cortex-a15_neon-vfpv4
+            target: armvirt-32
+            runtime_test: true
+
+          - arch: x86_64
+            target: x86-64
+            runtime_test: true
+
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v2
```

Comment:
v2 is deprecated. Why switch back from v3 to v2?

Reply:
I was not sure whether the new version can deal with the same recipe.